### PR TITLE
Worked on the logic for the carousels and a bit for the modal window

### PR DESCRIPTION
**Pull request by SebasCC99**
Hope it's not a lot 😬

### CODE REVIEW

File: client/src/related_items/Card.jsx [hunk 1]
```diff
@@ -4,15 +4,18 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faStar as solidStar } from '@fortawesome/free-solid-svg-icons'
 import { faStar as emptyStar } from '@fortawesome/free-regular-svg-icons'
 
+import ModalWindow from './ModalWindow';
 import helpers from './helpers';
 import './styles/Card.css';
 
 const Card = (props) => {
+  const currentOverviewId = props.currentId;
```
Comment (luna-moon-1216) [inline client/src/related_items/Card.jsx:12]: What's the difference between these 2 ids

File: client/src/related_items/Outfit.jsx [hunk 2]
```diff
@@ -1,16 +1,44 @@
...
+  const handleForward = () => {
+    // TODO: FIX LOGIC FOR OUTFIT CAROUSEL
+  }
+
+  const handleClick = () => {
+    if (outfits.includes(currentProductId)) return
+    setOutfits(outfits => [...outfits, currentProductId])
+  }
+
+  const displayProducts = (outfits.length > 3) ? outfits.slice(firstDisplayed, lastDisplayed) : outfits;
```
Comment (luna-moon-1216) [inline client/src/related_items/Outfit.jsx:25]: This is smart :)

File: client/src/related_items/Card.jsx (hunk 1 above)
Comment (SebasCC99) [reply client/src/related_items/Card.jsx:12]: One is the current product id in the overview section, the other one is the product id of the card being clicked. I can rename them for more clarity

File: client/src/related_items/Outfit.jsx (hunk 2 above)
Comment (SebasCC99) [reply client/src/related_items/Outfit.jsx:25]: Thank you!

**Review by luna-moon-1216 — APPROVED** [review]
💯

**Pull request merged**